crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -75,6 +63,7 @@ impl PwBackend {
         println!("pipewire backend running");
 
         Self {
+            stream_params : Vec::<PCMParams>::with_capacity(NR_STREAMS)
```

Comment:
No need for using `Vec::<T>` here, the compiler can figure out the type

Reply:
Thanks